plugins/lsp/language-servers/efmls-configs-pkgs.nix

```diff
@@ -27,115 +27,145 @@ pkgs: {
-      protolint
-      pylint
-      rubocop
-      ruff
-      rustfmt
-      scalafmt
-      selene
-      shellcheck
-      shellharden
-      shfmt
-      smlfmt
-      sqlfluff
-      statix
-      stylua
-      taplo
-      typstfmt
-      typstyle
-      uncrustify
-      vale
-      yamllint
-      yapf
-      ;
-    inherit (python3.pkgs)
-      autopep8
-      flake8
-      mdformat
-      vulture
-      ;
-    inherit (nodePackages)
-      eslint_d
-      prettier
-      alex
-      sql-formatter
-      stylelint
-      textlint
-      ;
-    inherit (phpPackages) phan phpstan psalm;
-    inherit (luaPackages) luacheck;
-    inherit (haskellPackages) fourmolu;
-    ansible_lint = ansible-lint;
-    chktex = texliveMedium;
-    clang_format = clang-tools;
-    clang_tidy = clang-tools;
-    clj_kondo = clj-kondo;
-    cmake_lint = cmake-format;
-    dartfmt = dart;
-    dotnet_format = dotnet-runtime;
-    # TODO: Added 2024-08-31; remove 2024-11-31
-    # eslint was moved out of nodePackages set without alias
-    # Using fallback as a transition period
-    eslint = pkgs.eslint or pkgs.nodePackages.eslint;
-    fish_indent = fish;
-    gofmt = go;
-    goimports = go-tools;
-    golangci_lint = golangci-lint;
-    google_java_format = google-java-format;
-    go_revive = revive;
-    latexindent = texliveMedium;
-    lua_format = luaformatter;
-    markdownlint = markdownlint-cli;
-    mcs = mono;
-    nixfmt = nixfmt-classic;
-    phpcbf = phpPackages.php-codesniffer;
-    php_cs_fixer = phpPackages.php-cs-fixer;
-    phpcs = phpPackages.php-codesniffer;
-    prettier_d = prettierd;
-    slither = slither-analyzer;
-    staticcheck = go-tools;
-    terraform_fmt = terraform;
-    vint = vim-vint;
-    write_good = write-good;
-    yq = yq-go;
-  };
+  packaged =
+    # Top-level packages
+    lib.genAttrs [
+      "actionlint"
+      "alejandra"
+      "ameba"
+      "astyle"
+      "bashate"
+      "beautysh"
+      "biome"
+      "black"
+      "buf"
+      "cbfmt"
+      "checkmake"
+      "clazy"
+      "codespell"
+      "cppcheck"
+      "cpplint"
+      "dfmt"
+      "djlint"
+      "dmd"
+      "dprint"
+      "fish"
+      "flawfinder"
+      "fnlfmt"
+      "gcc"
+      "gitlint"
+      "gofumpt"
+      "golines"
+      "golint"
+      "hadolint"
+      "isort"
+      "joker"
+      "jq"
+      "languagetool"
+      "mypy"
+      "php"
+      "prettypst"
+      "proselint"
+      "protolint"
+      "pylint"
+      "rubocop"
+      "ruff"
+      "rustfmt"
+      "scalafmt"
+      "selene"
+      "shellcheck"
+      "shellharden"
+      "shfmt"
+      "smlfmt"
+      "sqlfluff"
+      "statix"
+      "stylua"
+      "taplo"
+      "typstfmt"
+      "typstyle"
+      "uncrustify"
+      "vale"
+      "yamllint"
+      "yapf"
+    ] lib.id
+    # Scoped packages
+    //
+      lib.concatMapAttrs
+        (
+          scope: names:
+          lib.genAttrs names (name: [
+            scope
+            name
+          ])
```

Comment:
This does not seem right, it generates a list of two names, but that results in two packages named `nodePackges` and `eslint_d` trying to get installed for example.

I'm not sure what is the right syntax hele, `nodePackages.eslint_t` ?

Reply:
I'll have to look into it in the repl. The intended behaviour is to produce `[ "nodePackagse" "eslint_t" ]`, e.g:

```nix
{
  autopep8 = [ "python3" "autopep8" ];
  eslint_t = [ "nodePackagse" "eslint_t" ];
  prettier =  [ "nodePackagse" "prettier" ];
}
```